src/Sdk/DTWebApi/WebApi/AgentRefreshMessage.cs

```diff
@@ -45,5 +48,12 @@ public String TargetVersion
             get;
             private set;
         }
+
+        [DataMember]
+        public PackageMetadata PackageMetadata
```

Comment:
is this change in Actions service already?

Reply:
no, does it need to be?